testDb.prisma

```diff
@@ -0,0 +1,24 @@
+datasource db {
```

Comment:
Maybe move this files to a `tests` subfolder where more automated tests will live already?